### DISCUSSION TITLE
consider making a "Mass delete" of entities from a specific integration?

### DISCUSSION THREAD

**Discussion by phellarv** [frenck/spook#287] [Ideas]
Or not so much of a list, but I did enable the iBeacon tracker a while ago, and I live in an area with loads of people passing by.
So my iBeacon list is filled with entities which is unknow, and I do not need.
Can you consider making a "Mass delete" of entities from a spesific integration?

**Comment by frenck** [comment]
Why not disable the creation of new entities for that integration in that case?

![CleanShot 2023-08-10 at 16 31 18](https://github.com/frenck/spook/assets/195327/c60b1b56-d126-45ab-8d1b-88b40fe3bbb9)

**Comment by phellarv** [comment]
I stopped discovering now, and got all the cruft deleted. Thanks.
